source/triggers/database-triggers.txt

```diff
@@ -466,18 +466,19 @@ importing an application directory with {+cli-bin+}.
 Last Cluster Time Processed
 ---------------------------
 
-All Triggers keep track of the last time the Trigger executed, known as
-the **latest execution** in the App Services UI. Database Triggers also keep track
-of the **last cluster time processed**: the last time the change stream
-backing that Trigger emitted an event. You can access these timestamps
-in the list of Triggers that displays when you first navigate to the
+Atlas App Services keeps track of the last time a trigger was run. If the trigger 
+is not sending any events, the server sends a heartbeat to ensure the trigger's 
+resume token stays fresh. Whichever event is most recent is shown as the 
+:guilabel:`Latest Heartbeat` in the App Services UI. Database Triggers also 
+keep track of the :guilabel:`Last Cluster Time Processed`, which is the last time 
+the change stream backing that Trigger emitted an event. You can access these 
+timestamps in the list of Triggers that displays when you first navigate to the
 :guilabel:`Triggers` section from the sidebar.
 
-The difference between the latest execution and the last cluster time
+The difference between the latest heartbeat and the last cluster time
```

Comment:
I believe this second statement is no longer true. The heartbeat can be now and the last cluster time can be a day ago and that just means it has been 1 day since an event, right? 

cc @Calvinnix and @dpish for clarity

Reply:
Thanks. Because we define it (a bit incorrectly) in the paragraph above, I'm gong to clean this whole section up just a bit.

Reply:
@dpish 
You say "The heartbeat can be now(), it indicates the last time the trigger attempted to process any available events."
I thought that if there are no events, we send a heartbeat, and this value might just represent that, not just the last time an event was handled. 
As for the timing on Last Cluster Time, are you saying it can't be more than 1 day, or are you just using that as an example?

Reply:
In other words, is this correct? 
> **Latest Heartbeat**,

Atlas App Services keeps track of the last time a trigger was run. If the trigger 
is not sending any events, the server sends a heartbeat to ensure the trigger's 
resume token stays fresh. Whichever event is most recent is shown as the 
:guilabel:`Latest Heartbeat`.

**Last Cluster Time Processed**

This is the time when the trigger last processed an event. It will be older than 
the :guilabel:`Latest Heartbeat` if there have been no events since the most 
recent heartbeat.

Reply:
Last Heartbeat looks great.

For Last Cluster Time Processed, what you had earlier is correct:
> Database Triggers also keep track of the :guilabel:`Last Cluster Time Processed`, which is the last time 
> the change stream backing that Trigger emitted an event. 

So it could be something like:
> Database Triggers keep track of the :guilabel:`Last Cluster Time Processed`, which is the last time 
> the change stream backing that Trigger emitted an event. It will be older than
> the :guilabel:Latest Heartbeat if there have been no events since the most
> recent heartbeat.